packages/gasket-plugin-intl/lib/utils/get-preferred-locale.js

```diff
@@ -35,12 +35,12 @@ function getLocaleFromHeaders(gasket, req, locales, defaultLocale) {
   let preferredLocale = defaultLocale;
   /** @type {string} */
   const acceptLanguage = req.headers['accept-language'];
-
+  const negotiator = new Negotiator(req);
   if (acceptLanguage) {
     debug(`Received accept-language of ${acceptLanguage}`);
     try {
       // Get highest or highest from locales if configured
-      preferredLocale = formatLocale(accept.language(acceptLanguage, locales));
+      preferredLocale = formatLocale(negotiator.languages(locales)[0]);
```

Comment:
You could just use the [lanuage()](https://www.npmjs.com/package/negotiator#methods-1) method:
```suggestion
      preferredLocale = formatLocale(negotiator.language(locales));
```